### PR TITLE
ReactDOM: remove unstable-native-dependencies bundle

### PR DESCRIPTION
**Pull request by necolas**
This is only used by react-native-web and will be replaced by a user-space
implementation

## Summary

See: https://github.com/necolas/react-native-web/issues/1568

## Test Plan

CI

### CODE REVIEW

File: scripts/rollup/bundles.js [hunk 1]
```diff
@@ -155,15 +155,6 @@ const bundles = [
     externals: ['react'],
   },
 
-  /* React DOM internals required for react-native-web (e.g., to shim native events from react-dom) */
-  {
-    bundleTypes: [UMD_DEV, UMD_PROD, NODE_DEV, NODE_PROD],
-    moduleType: RENDERER_UTILS,
-    entry: 'react-dom/unstable-native-dependencies',
```
[the remark this answers is not in the document]
Comment (gaearon) [reply scripts/rollup/bundles.js:162]: And the one they re-export.

[the remark this answers is not in the document]
Comment (gaearon) [reply scripts/rollup/bundles.js:162]: And also `package.json` `"files"` entry.

[the remark this answers is not in the document]
Comment (necolas) [reply scripts/rollup/bundles.js:162]: I think these files are the only place the `ResponderEventPlugin` is actually tested, including for React Native

**Comment by necolas** [comment]
>  I think it'll also have to wait until we're finished releasing minors and are ready for a major

Probably. I'm expecting to get the replacement into RNW by the end of the month. This doesn't strictly need to go out in a major, since previous breaking changes in these files have occurred in patch releases. However, if there's anything else related to the plugin system that we want to remove once this is merged, then it would make sense to package it all up in a major.

**Comment by necolas** [comment]
I'm happy to merge this and will be responsible for managing the downstream effects.

**Pull request merged**